[2024-10-11T18:46:24.774+0000] {processor.py:153} INFO - Started process (PID=64) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:46:24.775+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:46:24.776+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:46:24.776+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:46:24.792+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:46:24.791+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 4, in <module>
    from scripts.fetch_data import fetch_games, fetch_players, fetch_teams, fetch_stats
ModuleNotFoundError: No module named 'scripts'
[2024-10-11T18:46:24.792+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:46:24.833+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.063 seconds
[2024-10-11T18:46:55.296+0000] {processor.py:153} INFO - Started process (PID=110) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:46:55.297+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:46:55.298+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:46:55.297+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:46:55.304+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:46:55.304+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 4, in <module>
    from scripts.fetch_data import fetch_games, fetch_players, fetch_teams, fetch_stats
ModuleNotFoundError: No module named 'scripts'
[2024-10-11T18:46:55.305+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:46:55.316+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.023 seconds
[2024-10-11T18:47:25.646+0000] {processor.py:153} INFO - Started process (PID=156) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:47:25.647+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:47:25.648+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:47:25.648+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:47:25.662+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:47:25.661+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 4, in <module>
    from scripts.fetch_data import fetch_games, fetch_players, fetch_teams, fetch_stats
ModuleNotFoundError: No module named 'scripts'
[2024-10-11T18:47:25.663+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:47:25.682+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.041 seconds
[2024-10-11T18:47:30.728+0000] {processor.py:153} INFO - Started process (PID=175) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:47:30.729+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:47:30.730+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:47:30.730+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:47:30.743+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:47:30.742+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 25, in <module>
    python_callable=create_tables,
NameError: name 'create_tables' is not defined
[2024-10-11T18:47:30.743+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:47:30.755+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.032 seconds
[2024-10-11T18:47:33.798+0000] {processor.py:153} INFO - Started process (PID=176) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:47:33.799+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:47:33.800+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:47:33.800+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:47:33.805+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:47:33.804+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 4
    from
        ^
SyntaxError: invalid syntax
[2024-10-11T18:47:33.805+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:47:33.826+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.033 seconds
[2024-10-11T18:47:40.872+0000] {processor.py:153} INFO - Started process (PID=195) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:47:40.873+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:47:40.875+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:47:40.875+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:47:40.882+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:47:40.879+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 4
    from scr
           ^
SyntaxError: invalid syntax
[2024-10-11T18:47:40.883+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:47:40.899+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.031 seconds
[2024-10-11T18:47:43.943+0000] {processor.py:153} INFO - Started process (PID=196) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:47:43.945+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:47:43.946+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:47:43.946+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:47:43.963+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:47:43.962+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 4, in <module>
    from scripts.create_tables import create_tables
ModuleNotFoundError: No module named 'scripts'
[2024-10-11T18:47:43.963+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:47:43.979+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.043 seconds
[2024-10-11T18:47:46.008+0000] {processor.py:153} INFO - Started process (PID=197) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:47:46.008+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:47:46.009+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:47:46.009+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:47:46.016+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:47:46.016+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 4, in <module>
    from scripts.create_tables import create_tables
ModuleNotFoundError: No module named 'scripts'
[2024-10-11T18:47:46.017+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:47:46.029+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.024 seconds
[2024-10-11T18:47:50.090+0000] {processor.py:153} INFO - Started process (PID=198) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:47:50.091+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:47:50.092+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:47:50.092+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:47:50.110+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:47:50.107+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 4, in <module>
    from scripts.create_tables import create_tables
ModuleNotFoundError: No module named 'scripts'
[2024-10-11T18:47:50.112+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:47:50.141+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.058 seconds
[2024-10-11T18:47:53.199+0000] {processor.py:153} INFO - Started process (PID=217) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:47:53.201+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:47:53.202+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:47:53.202+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:47:53.216+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:47:53.215+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 4, in <module>
    from scripts.create_tables import create_tables
ModuleNotFoundError: No module named 'scripts'
[2024-10-11T18:47:53.216+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:47:53.235+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.042 seconds
[2024-10-11T18:47:58.306+0000] {processor.py:153} INFO - Started process (PID=218) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:47:58.309+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:47:58.310+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:47:58.310+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:47:58.315+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:47:58.314+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 9
    from scripts.insert_data import insert_teams, insert_players, insert_games,
                                   ^
SyntaxError: trailing comma not allowed without surrounding parentheses
[2024-10-11T18:47:58.316+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:47:58.336+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.036 seconds
[2024-10-11T18:48:00.339+0000] {processor.py:153} INFO - Started process (PID=219) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:48:00.341+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:48:00.342+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:48:00.342+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:48:00.355+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:48:00.354+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 4, in <module>
    from scripts.create_tables import create_tables
ModuleNotFoundError: No module named 'scripts'
[2024-10-11T18:48:00.356+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:48:00.374+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.040 seconds
[2024-10-11T18:48:34.416+0000] {processor.py:153} INFO - Started process (PID=44) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:48:34.418+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:48:34.418+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:48:34.418+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:48:34.424+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:48:34.423+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 4, in <module>
    from scripts.create_tables import create_tables
ModuleNotFoundError: No module named 'scripts'
[2024-10-11T18:48:34.425+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:48:34.437+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.026 seconds
[2024-10-11T18:48:56.627+0000] {processor.py:153} INFO - Started process (PID=81) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:48:56.629+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:48:56.630+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:48:56.630+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:48:56.639+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:48:56.638+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 4, in <module>
    from .scripts.create_tables import create_tables
ImportError: attempted relative import with no known parent package
[2024-10-11T18:48:56.639+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:48:56.662+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.038 seconds
[2024-10-11T18:49:26.857+0000] {processor.py:153} INFO - Started process (PID=136) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:49:26.858+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:49:26.859+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:49:26.859+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:49:26.866+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:49:26.865+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 4, in <module>
    from .scripts.create_tables import create_tables
ImportError: attempted relative import with no known parent package
[2024-10-11T18:49:26.866+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:49:26.878+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.024 seconds
[2024-10-11T18:49:43.045+0000] {processor.py:153} INFO - Started process (PID=155) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:49:43.047+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:49:43.048+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:49:43.048+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:49:43.061+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:49:43.060+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 4, in <module>
    from data_pipeline.scripts.create_tables import create_tables
ModuleNotFoundError: No module named 'data_pipeline'
[2024-10-11T18:49:43.061+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:49:43.080+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.039 seconds
[2024-10-11T18:50:10.259+0000] {processor.py:153} INFO - Started process (PID=44) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:50:10.260+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:50:10.260+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:50:10.260+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:50:10.266+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:50:10.265+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 4, in <module>
    from data_pipeline.scripts.create_tables import create_tables
ModuleNotFoundError: No module named 'data_pipeline'
[2024-10-11T18:50:10.266+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:50:10.277+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.021 seconds
[2024-10-11T18:50:40.535+0000] {processor.py:153} INFO - Started process (PID=99) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:50:40.536+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:50:40.538+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:50:40.537+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:50:40.551+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:50:40.549+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 4, in <module>
    from data_pipeline.scripts.create_tables import create_tables
ModuleNotFoundError: No module named 'data_pipeline'
[2024-10-11T18:50:40.551+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:50:40.568+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.039 seconds
[2024-10-11T18:51:10.821+0000] {processor.py:153} INFO - Started process (PID=145) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:51:10.822+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:51:10.823+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:51:10.822+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:51:10.831+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:51:10.830+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 4, in <module>
    from data_pipeline.scripts.create_tables import create_tables
ModuleNotFoundError: No module named 'data_pipeline'
[2024-10-11T18:51:10.832+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:51:10.847+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.029 seconds
[2024-10-11T18:51:39.602+0000] {processor.py:153} INFO - Started process (PID=44) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:51:39.604+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:51:39.606+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:51:39.605+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:51:39.617+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:51:39.616+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 4, in <module>
    from data_pipeline.scripts.create_tables import create_tables
ModuleNotFoundError: No module named 'data_pipeline'
[2024-10-11T18:51:39.618+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:51:39.630+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.032 seconds
[2024-10-11T18:52:09.952+0000] {processor.py:153} INFO - Started process (PID=100) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:52:09.954+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:52:09.955+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:52:09.955+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:52:09.968+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:52:09.967+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 4, in <module>
    from data_pipeline.scripts.create_tables import create_tables
ModuleNotFoundError: No module named 'data_pipeline'
[2024-10-11T18:52:09.969+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:52:09.986+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.039 seconds
[2024-10-11T18:52:22.074+0000] {processor.py:153} INFO - Started process (PID=119) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:52:22.076+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:52:22.077+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:52:22.077+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:52:22.095+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:52:22.093+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 4, in <module>
    from data_pipeline.scripts.create_tables import create_tables
ModuleNotFoundError: No module named 'data_pipeline'
[2024-10-11T18:52:22.096+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:52:22.119+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.049 seconds
[2024-10-11T18:52:27.149+0000] {processor.py:153} INFO - Started process (PID=120) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:52:27.152+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:52:27.153+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:52:27.153+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:52:27.167+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:52:27.166+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 4, in <module>
    from data_pipeline.scripts.create_tables import create_tables
ModuleNotFoundError: No module named 'data_pipeline'
[2024-10-11T18:52:27.168+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:52:27.192+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.048 seconds
[2024-10-11T18:53:56.096+0000] {processor.py:153} INFO - Started process (PID=44) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:53:56.097+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:53:56.097+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:53:56.097+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:53:56.105+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:53:56.104+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 4, in <module>
    from scripts.create_tables import create_tables
ModuleNotFoundError: No module named 'scripts'
[2024-10-11T18:53:56.105+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:53:56.117+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.024 seconds
[2024-10-11T18:54:26.397+0000] {processor.py:153} INFO - Started process (PID=99) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:54:26.398+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:54:26.399+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:54:26.399+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:54:26.424+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:54:26.423+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 4, in <module>
    from scripts.create_tables import create_tables
ModuleNotFoundError: No module named 'scripts'
[2024-10-11T18:54:26.425+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:54:26.469+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.075 seconds
[2024-10-11T18:54:56.725+0000] {processor.py:153} INFO - Started process (PID=154) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:54:56.726+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:54:56.726+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:54:56.726+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:54:56.733+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:54:56.732+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 4, in <module>
    from scripts.create_tables import create_tables
ModuleNotFoundError: No module named 'scripts'
[2024-10-11T18:54:56.733+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:54:56.748+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.026 seconds
[2024-10-11T18:54:57.768+0000] {processor.py:153} INFO - Started process (PID=155) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:54:57.769+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:54:57.769+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:54:57.769+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:54:57.779+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:54:57.778+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 4, in <module>
    from ..scripts.create_tables import create_tables
ImportError: attempted relative import with no known parent package
[2024-10-11T18:54:57.779+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:54:57.790+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.040 seconds
[2024-10-11T18:55:01.809+0000] {processor.py:153} INFO - Started process (PID=156) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:55:01.810+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:55:01.811+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:55:01.811+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:55:01.821+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:55:01.820+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 4, in <module>
    from ..create_tablesscripts.create_tables import create_tables
ImportError: attempted relative import with no known parent package
[2024-10-11T18:55:01.822+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:55:01.840+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.034 seconds
[2024-10-11T18:55:07.885+0000] {processor.py:153} INFO - Started process (PID=175) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:55:07.885+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:55:07.886+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:55:07.886+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:55:07.889+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:55:07.888+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 4
    from
        ^
SyntaxError: invalid syntax
[2024-10-11T18:55:07.889+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:55:07.899+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.017 seconds
[2024-10-11T18:55:08.899+0000] {processor.py:153} INFO - Started process (PID=176) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:55:08.900+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:55:08.901+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:55:08.901+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:55:08.904+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:55:08.904+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 4
    from ..
          ^
SyntaxError: invalid syntax
[2024-10-11T18:55:08.905+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:55:08.918+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.022 seconds
[2024-10-11T18:55:10.953+0000] {processor.py:153} INFO - Started process (PID=177) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:55:10.955+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:55:10.956+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:55:10.956+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:55:10.962+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:55:10.961+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 4
    from ..s
           ^
SyntaxError: invalid syntax
[2024-10-11T18:55:10.963+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:55:10.981+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.033 seconds
[2024-10-11T18:55:12.985+0000] {processor.py:153} INFO - Started process (PID=178) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:55:12.987+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:55:12.988+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:55:12.987+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:55:12.998+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:55:12.997+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 4, in <module>
    from ..scripts.create_tables import create_tables
ImportError: attempted relative import with no known parent package
[2024-10-11T18:55:12.998+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:55:13.014+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.033 seconds
[2024-10-11T18:55:20.061+0000] {processor.py:153} INFO - Started process (PID=197) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:55:20.062+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:55:20.063+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:55:20.063+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:55:20.067+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:55:20.066+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 5
    from
        ^
SyntaxError: invalid syntax
[2024-10-11T18:55:20.068+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:55:20.087+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.029 seconds
[2024-10-11T18:55:32.197+0000] {processor.py:153} INFO - Started process (PID=216) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:55:32.200+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:55:32.201+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:55:32.201+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:55:32.207+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:55:32.206+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 5
    from ..scripts.fetch_data
                            ^
SyntaxError: invalid syntax
[2024-10-11T18:55:32.208+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:55:32.228+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.035 seconds
[2024-10-11T18:55:34.215+0000] {processor.py:153} INFO - Started process (PID=217) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:55:34.215+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:55:34.216+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:55:34.216+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:55:34.225+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:55:34.224+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 4, in <module>
    from ..scripts.create_tables import create_tables
ImportError: attempted relative import with no known parent package
[2024-10-11T18:55:34.226+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:55:34.237+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.025 seconds
[2024-10-11T18:55:36.243+0000] {processor.py:153} INFO - Started process (PID=218) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:55:36.245+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:55:36.245+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:55:36.245+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:55:36.256+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:55:36.255+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 4, in <module>
    from ..scripts.create_tables import create_tables
ImportError: attempted relative import with no known parent package
[2024-10-11T18:55:36.257+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:55:36.275+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.037 seconds
[2024-10-11T18:56:06.673+0000] {processor.py:153} INFO - Started process (PID=273) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:56:06.674+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:56:06.675+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:56:06.675+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:56:06.683+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:56:06.682+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 4, in <module>
    from ..scripts.create_tables import create_tables
ImportError: attempted relative import with no known parent package
[2024-10-11T18:56:06.684+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:56:06.699+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.030 seconds
[2024-10-11T18:56:36.986+0000] {processor.py:153} INFO - Started process (PID=310) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:56:36.987+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:56:36.988+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:56:36.988+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:56:37.001+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:56:37.000+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 4, in <module>
    from ..scripts.create_tables import create_tables
ImportError: attempted relative import with no known parent package
[2024-10-11T18:56:37.002+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:56:37.018+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.038 seconds
[2024-10-11T18:57:07.303+0000] {processor.py:153} INFO - Started process (PID=366) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:57:07.305+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:57:07.306+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:57:07.306+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:57:07.317+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:57:07.315+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 4, in <module>
    from ..scripts.create_tables import create_tables
ImportError: attempted relative import with no known parent package
[2024-10-11T18:57:07.317+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:57:07.336+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.039 seconds
[2024-10-11T18:57:37.629+0000] {processor.py:153} INFO - Started process (PID=421) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:57:37.632+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:57:37.633+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:57:37.633+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:57:37.646+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:57:37.644+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 4, in <module>
    from ..scripts.create_tables import create_tables
ImportError: attempted relative import with no known parent package
[2024-10-11T18:57:37.646+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:57:37.665+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.042 seconds
[2024-10-11T18:58:07.918+0000] {processor.py:153} INFO - Started process (PID=476) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:58:07.923+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:58:07.923+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:58:07.923+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:58:07.930+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:58:07.930+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 4, in <module>
    from ..scripts.create_tables import create_tables
ImportError: attempted relative import with no known parent package
[2024-10-11T18:58:07.931+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:58:07.942+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.030 seconds
[2024-10-11T18:58:38.248+0000] {processor.py:153} INFO - Started process (PID=513) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:58:38.249+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:58:38.249+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:58:38.249+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:58:38.255+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:58:38.254+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 4, in <module>
    from ..scripts.create_tables import create_tables
ImportError: attempted relative import with no known parent package
[2024-10-11T18:58:38.255+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:58:38.265+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.020 seconds
[2024-10-11T18:59:08.628+0000] {processor.py:153} INFO - Started process (PID=568) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:59:08.630+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:59:08.631+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:59:08.631+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:59:08.643+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:59:08.642+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 4, in <module>
    from ..scripts.create_tables import create_tables
ImportError: attempted relative import with no known parent package
[2024-10-11T18:59:08.644+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:59:08.662+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.039 seconds
[2024-10-11T18:59:10.647+0000] {processor.py:153} INFO - Started process (PID=569) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:59:10.647+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:59:10.649+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:59:10.648+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:59:10.655+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:59:10.655+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 6, in <module>
    from ..scripts.create_tables import create_tables
ImportError: attempted relative import with no known parent package
[2024-10-11T18:59:10.656+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:59:10.667+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.023 seconds
[2024-10-11T18:59:13.698+0000] {processor.py:153} INFO - Started process (PID=570) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:59:13.700+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:59:13.702+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:59:13.702+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:59:13.715+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:59:13.713+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 6, in <module>
    from ..scripts.create_tables import create_tables
ImportError: attempted relative import with no known parent package
[2024-10-11T18:59:13.716+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:59:13.737+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.046 seconds
[2024-10-11T18:59:20.766+0000] {processor.py:153} INFO - Started process (PID=589) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:59:20.766+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T18:59:20.767+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:59:20.767+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:59:20.777+0000] {logging_mixin.py:137} INFO - [2024-10-11T18:59:20.776+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 6, in <module>
    from scripts.create_tables import create_tables
  File "/opt/airflow/dags/scripts/create_tables.py", line 1, in <module>
    import pyscopg2
ModuleNotFoundError: No module named 'pyscopg2'
[2024-10-11T18:59:20.777+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T18:59:20.789+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.025 seconds
[2024-10-11T19:02:10.535+0000] {processor.py:153} INFO - Started process (PID=44) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:02:10.536+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T19:02:10.537+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:02:10.537+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:02:10.543+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:02:10.542+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 6, in <module>
    from scripts.create_tables import create_tables
  File "/opt/airflow/dags/scripts/create_tables.py", line 1, in <module>
    import pyscopg2
ModuleNotFoundError: No module named 'pyscopg2'
[2024-10-11T19:02:10.544+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:02:10.555+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.022 seconds
[2024-10-11T19:02:40.842+0000] {processor.py:153} INFO - Started process (PID=99) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:02:40.844+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T19:02:40.845+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:02:40.845+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:02:40.859+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:02:40.858+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 6, in <module>
    from scripts.create_tables import create_tables
  File "/opt/airflow/dags/scripts/create_tables.py", line 1, in <module>
    import pyscopg2
ModuleNotFoundError: No module named 'pyscopg2'
[2024-10-11T19:02:40.860+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:02:40.873+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.036 seconds
[2024-10-11T19:03:11.151+0000] {processor.py:153} INFO - Started process (PID=144) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:03:11.152+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T19:03:11.153+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:03:11.153+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:03:11.162+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:03:11.161+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 6, in <module>
    from scripts.create_tables import create_tables
  File "/opt/airflow/dags/scripts/create_tables.py", line 1, in <module>
    import pyscopg2
ModuleNotFoundError: No module named 'pyscopg2'
[2024-10-11T19:03:11.162+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:03:11.173+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.024 seconds
[2024-10-11T19:03:41.394+0000] {processor.py:153} INFO - Started process (PID=190) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:03:41.395+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T19:03:41.396+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:03:41.396+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:03:41.406+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:03:41.405+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 6, in <module>
    from scripts.create_tables import create_tables
  File "/opt/airflow/dags/scripts/create_tables.py", line 1, in <module>
    import pyscopg2
ModuleNotFoundError: No module named 'pyscopg2'
[2024-10-11T19:03:41.406+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:03:41.421+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.031 seconds
[2024-10-11T19:11:04.149+0000] {processor.py:153} INFO - Started process (PID=64) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:11:04.151+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T19:11:04.152+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:11:04.152+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:11:04.166+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:11:04.165+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 6, in <module>
    from scripts.create_tables import create_tables
  File "/opt/airflow/dags/scripts/create_tables.py", line 1, in <module>
    import pyscopg2
ModuleNotFoundError: No module named 'pyscopg2'
[2024-10-11T19:11:04.168+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:11:04.184+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.038 seconds
[2024-10-11T19:11:34.526+0000] {processor.py:153} INFO - Started process (PID=119) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:11:34.536+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T19:11:34.551+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:11:34.551+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:11:34.562+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:11:34.561+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 6, in <module>
    from scripts.create_tables import create_tables
  File "/opt/airflow/dags/scripts/create_tables.py", line 1, in <module>
    import pyscopg2
ModuleNotFoundError: No module named 'pyscopg2'
[2024-10-11T19:11:34.563+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:11:34.624+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.108 seconds
[2024-10-11T19:12:04.829+0000] {processor.py:153} INFO - Started process (PID=156) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:12:04.830+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T19:12:04.831+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:12:04.831+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:12:05.324+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:12:05.321+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 8, in <module>
    from scripts.insert_data import insert_teams, insert_players, insert_games, insert_player_stats
  File "/opt/airflow/dags/scripts/insert_data.py", line 1, in <module>
    from config import get_db_connection
ModuleNotFoundError: No module named 'config'
[2024-10-11T19:12:05.325+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:12:05.339+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.515 seconds
[2024-10-11T19:12:35.654+0000] {processor.py:153} INFO - Started process (PID=211) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:12:35.656+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T19:12:35.657+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:12:35.657+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:12:35.872+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:12:35.869+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 8, in <module>
    from scripts.insert_data import insert_teams, insert_players, insert_games, insert_player_stats
  File "/opt/airflow/dags/scripts/insert_data.py", line 1, in <module>
    from config import get_db_connection
ModuleNotFoundError: No module named 'config'
[2024-10-11T19:12:35.872+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:12:35.887+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.240 seconds
[2024-10-11T19:13:05.937+0000] {processor.py:153} INFO - Started process (PID=267) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:13:05.940+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T19:13:05.941+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:13:05.941+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:13:06.098+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:13:06.095+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 8, in <module>
    from scripts.insert_data import insert_teams, insert_players, insert_games, insert_player_stats
  File "/opt/airflow/dags/scripts/insert_data.py", line 1, in <module>
    from config import get_db_connection
ModuleNotFoundError: No module named 'config'
[2024-10-11T19:13:06.098+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:13:06.109+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.178 seconds
[2024-10-11T19:13:36.700+0000] {processor.py:153} INFO - Started process (PID=305) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:13:36.702+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T19:13:36.704+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:13:36.704+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:13:36.912+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:13:36.908+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 8, in <module>
    from scripts.insert_data import insert_teams, insert_players, insert_games, insert_player_stats
  File "/opt/airflow/dags/scripts/insert_data.py", line 1, in <module>
    from config import get_db_connection
ModuleNotFoundError: No module named 'config'
[2024-10-11T19:13:36.912+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:13:36.926+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.232 seconds
[2024-10-11T19:14:07.001+0000] {processor.py:153} INFO - Started process (PID=361) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:14:07.003+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T19:14:07.004+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:14:07.004+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:14:07.023+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:14:07.021+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 6, in <module>
    from scripts.create_tables import create_tables
  File "/opt/airflow/dags/scripts/create_tables.py", line 1, in <module>
    import pyscopg2
ModuleNotFoundError: No module named 'pyscopg2'
[2024-10-11T19:14:07.023+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:14:07.043+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.048 seconds
[2024-10-11T19:18:01.904+0000] {processor.py:153} INFO - Started process (PID=65) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:18:01.905+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T19:18:01.906+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:18:01.906+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:18:02.454+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:18:02.452+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 8, in <module>
    from scripts.insert_data import insert_teams, insert_players, insert_games, insert_player_stats
  File "/opt/airflow/dags/scripts/insert_data.py", line 1, in <module>
    from config import get_db_connection
ModuleNotFoundError: No module named 'config'
[2024-10-11T19:18:02.455+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:18:02.857+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.955 seconds
[2024-10-11T19:18:33.165+0000] {processor.py:153} INFO - Started process (PID=120) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:18:33.166+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T19:18:33.167+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:18:33.167+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:18:33.390+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:18:33.385+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 8, in <module>
    from scripts.insert_data import insert_teams, insert_players, insert_games, insert_player_stats
  File "/opt/airflow/dags/scripts/insert_data.py", line 1, in <module>
    from config import get_db_connection
ModuleNotFoundError: No module named 'config'
[2024-10-11T19:18:33.390+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:18:33.404+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.242 seconds
[2024-10-11T19:19:03.445+0000] {processor.py:153} INFO - Started process (PID=158) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:19:03.447+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T19:19:03.448+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:19:03.448+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:19:03.602+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:19:03.599+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 8, in <module>
    from scripts.insert_data import insert_teams, insert_players, insert_games, insert_player_stats
  File "/opt/airflow/dags/scripts/insert_data.py", line 1, in <module>
    from config import get_db_connection
ModuleNotFoundError: No module named 'config'
[2024-10-11T19:19:03.602+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:19:03.612+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.172 seconds
[2024-10-11T19:19:34.196+0000] {processor.py:153} INFO - Started process (PID=213) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:19:34.198+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T19:19:34.199+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:19:34.199+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:19:34.394+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:19:34.392+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 8, in <module>
    from scripts.insert_data import insert_teams, insert_players, insert_games, insert_player_stats
  File "/opt/airflow/dags/scripts/insert_data.py", line 1, in <module>
    from config import get_db_connection
ImportError: cannot import name 'get_db_connection' from 'config' (/opt/airflow/dags/config/__init__.py)
[2024-10-11T19:19:34.394+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:19:34.407+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.218 seconds
[2024-10-11T19:19:46.527+0000] {processor.py:153} INFO - Started process (PID=233) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:19:46.529+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T19:19:46.529+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:19:46.529+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:19:46.719+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:19:46.715+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 9, in <module>
    from scripts.insert_data import insert_teams, insert_players, insert_games, insert_player_stats
  File "/opt/airflow/dags/scripts/insert_data.py", line 1, in <module>
    from config import get_db_connection
ImportError: cannot import name 'get_db_connection' from 'config' (/opt/airflow/dags/config/__init__.py)
[2024-10-11T19:19:46.719+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:19:46.734+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.215 seconds
[2024-10-11T19:19:50.884+0000] {processor.py:153} INFO - Started process (PID=235) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:19:50.886+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T19:19:50.887+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:19:50.886+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:19:51.103+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:19:51.100+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 9, in <module>
    from scripts.insert_data import insert_teams, insert_players, insert_games, insert_player_stats
  File "/opt/airflow/dags/scripts/insert_data.py", line 1, in <module>
    from config import get_db_connection
ImportError: cannot import name 'get_db_connection' from 'config' (/opt/airflow/dags/config/__init__.py)
[2024-10-11T19:19:51.104+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:19:51.125+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.247 seconds
[2024-10-11T19:19:53.907+0000] {processor.py:153} INFO - Started process (PID=255) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:19:53.909+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T19:19:53.910+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:19:53.910+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:19:54.056+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:19:54.054+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 9, in <module>
    from scripts.insert_data import insert_teams, insert_players, insert_games, insert_player_stats
  File "/opt/airflow/dags/scripts/insert_data.py", line 1, in <module>
    from config import get_db_connection
ImportError: cannot import name 'get_db_connection' from 'config' (/opt/airflow/dags/config/__init__.py)
[2024-10-11T19:19:54.057+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:19:54.066+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.165 seconds
[2024-10-11T19:19:56.121+0000] {processor.py:153} INFO - Started process (PID=257) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:19:56.123+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T19:19:56.124+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:19:56.124+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:19:56.260+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:19:56.258+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 9, in <module>
    from scripts.insert_data import insert_teams, insert_players, insert_games, insert_player_stats
  File "/opt/airflow/dags/scripts/insert_data.py", line 1, in <module>
    from config import get_db_connection
ImportError: cannot import name 'get_db_connection' from 'config' (/opt/airflow/dags/config/__init__.py)
[2024-10-11T19:19:56.260+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:19:56.269+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.153 seconds
[2024-10-11T19:20:01.348+0000] {processor.py:153} INFO - Started process (PID=259) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:20:01.350+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T19:20:01.351+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:20:01.351+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:20:01.491+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:20:01.489+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 9, in <module>
    from scripts.insert_data import insert_teams, insert_players, insert_games, insert_player_stats
  File "/opt/airflow/dags/scripts/insert_data.py", line 1, in <module>
    from config import get_db_connection
ImportError: cannot import name 'get_db_connection' from 'config' (/opt/airflow/dags/config/__init__.py)
[2024-10-11T19:20:01.491+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:20:01.500+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.159 seconds
[2024-10-11T19:20:31.621+0000] {processor.py:153} INFO - Started process (PID=315) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:20:31.625+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T19:20:31.627+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:20:31.627+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:20:31.839+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:20:31.836+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 9, in <module>
    from scripts.insert_data import insert_teams, insert_players, insert_games, insert_player_stats
  File "/opt/airflow/dags/scripts/insert_data.py", line 1, in <module>
    from ..config import get_db_connection
ValueError: attempted relative import beyond top-level package
[2024-10-11T19:20:31.839+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:20:31.852+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.238 seconds
[2024-10-11T19:21:02.362+0000] {processor.py:153} INFO - Started process (PID=371) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:21:02.364+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T19:21:02.364+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:21:02.364+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:21:02.592+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:21:02.589+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 9, in <module>
    from scripts.insert_data import insert_teams, insert_players, insert_games, insert_player_stats
  File "/opt/airflow/dags/scripts/insert_data.py", line 3, in <module>
    from config import get_db_connection
ImportError: cannot import name 'get_db_connection' from 'config' (/opt/airflow/dags/config/__init__.py)
[2024-10-11T19:21:02.592+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:21:02.604+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.247 seconds
[2024-10-11T19:21:32.862+0000] {processor.py:153} INFO - Started process (PID=409) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:21:32.864+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T19:21:32.865+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:21:32.865+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:21:33.053+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:21:33.051+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/nba_data_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/nba_data_pipeline.py", line 9, in <module>
    from scripts.insert_data import insert_teams, insert_players, insert_games, insert_player_stats
  File "/opt/airflow/dags/scripts/insert_data.py", line 3, in <module>
    from config import get_db_connection
ImportError: cannot import name 'get_db_connection' from 'config' (/opt/airflow/dags/config/__init__.py)
[2024-10-11T19:21:33.055+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:21:33.080+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.223 seconds
[2024-10-11T19:22:03.539+0000] {processor.py:153} INFO - Started process (PID=465) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:22:03.540+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T19:22:03.541+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:22:03.541+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:22:03.777+0000] {processor.py:753} INFO - DAG(s) dict_keys(['nba_data_pipeline']) retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:22:05.147+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:22:05.147+0000] {manager.py:504} INFO - Created Permission View: can delete on DAG:nba_data_pipeline
[2024-10-11T19:22:05.150+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:22:05.150+0000] {manager.py:504} INFO - Created Permission View: can read on DAG:nba_data_pipeline
[2024-10-11T19:22:05.153+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:22:05.153+0000] {manager.py:504} INFO - Created Permission View: can edit on DAG:nba_data_pipeline
[2024-10-11T19:22:05.153+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:22:05.153+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2024-10-11T19:22:05.159+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:22:05.159+0000] {dag.py:2711} INFO - Creating ORM DAG for nba_data_pipeline
[2024-10-11T19:22:05.165+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:22:05.165+0000] {dag.py:3441} INFO - Setting next_dagrun for nba_data_pipeline to 2023-10-01T00:00:00+00:00, run_after=2023-10-02T00:00:00+00:00
[2024-10-11T19:22:05.175+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 1.646 seconds
[2024-10-11T19:22:43.882+0000] {processor.py:153} INFO - Started process (PID=45) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:22:43.883+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T19:22:43.883+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:22:43.883+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:22:44.095+0000] {processor.py:753} INFO - DAG(s) dict_keys(['nba_data_pipeline']) retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:22:44.137+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:22:44.137+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2024-10-11T19:22:44.149+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:22:44.149+0000] {dag.py:3441} INFO - Setting next_dagrun for nba_data_pipeline to 2023-10-01T00:00:00+00:00, run_after=2023-10-02T00:00:00+00:00
[2024-10-11T19:22:44.313+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.434 seconds
[2024-10-11T19:23:15.160+0000] {processor.py:153} INFO - Started process (PID=102) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:23:15.161+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T19:23:15.163+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:23:15.162+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:23:15.572+0000] {processor.py:753} INFO - DAG(s) dict_keys(['nba_data_pipeline']) retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:23:15.592+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:23:15.592+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2024-10-11T19:23:15.614+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:23:15.614+0000] {dag.py:3441} INFO - Setting next_dagrun for nba_data_pipeline to 2023-10-01T00:00:00+00:00, run_after=2023-10-02T00:00:00+00:00
[2024-10-11T19:23:15.909+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.753 seconds
[2024-10-11T19:23:46.516+0000] {processor.py:153} INFO - Started process (PID=154) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:23:46.522+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T19:23:46.527+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:23:46.526+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:23:46.895+0000] {processor.py:753} INFO - DAG(s) dict_keys(['nba_data_pipeline']) retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:23:46.920+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:23:46.920+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2024-10-11T19:23:46.960+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.451 seconds
[2024-10-11T19:24:17.599+0000] {processor.py:153} INFO - Started process (PID=202) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:24:17.602+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T19:24:17.603+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:24:17.603+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:24:17.770+0000] {processor.py:753} INFO - DAG(s) dict_keys(['nba_data_pipeline']) retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T19:24:17.784+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:24:17.784+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2024-10-11T19:24:17.797+0000] {logging_mixin.py:137} INFO - [2024-10-11T19:24:17.797+0000] {dag.py:3441} INFO - Setting next_dagrun for nba_data_pipeline to 2023-10-16T00:00:00+00:00, run_after=2023-10-17T00:00:00+00:00
[2024-10-11T19:24:17.805+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 0.211 seconds
[2024-10-11T21:49:49.908+0000] {processor.py:153} INFO - Started process (PID=45) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T21:49:49.910+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T21:49:49.911+0000] {logging_mixin.py:137} INFO - [2024-10-11T21:49:49.911+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T21:49:50.525+0000] {logging_mixin.py:137} INFO - [2024-10-11T21:49:50.525+0000] {taskmixin.py:205} WARNING - Dependency <Task(PythonOperator): fetch_teams>, insert_teams already registered for DAG: nba_data_pipeline
[2024-10-11T21:49:50.526+0000] {logging_mixin.py:137} INFO - [2024-10-11T21:49:50.526+0000] {taskmixin.py:205} WARNING - Dependency <Task(PythonOperator): insert_teams>, fetch_teams already registered for DAG: nba_data_pipeline
[2024-10-11T21:49:50.526+0000] {logging_mixin.py:137} INFO - [2024-10-11T21:49:50.526+0000] {taskmixin.py:205} WARNING - Dependency <Task(PythonOperator): fetch_players>, insert_players already registered for DAG: nba_data_pipeline
[2024-10-11T21:49:50.526+0000] {logging_mixin.py:137} INFO - [2024-10-11T21:49:50.526+0000] {taskmixin.py:205} WARNING - Dependency <Task(PythonOperator): insert_players>, fetch_players already registered for DAG: nba_data_pipeline
[2024-10-11T21:49:50.527+0000] {logging_mixin.py:137} INFO - [2024-10-11T21:49:50.527+0000] {taskmixin.py:205} WARNING - Dependency <Task(PythonOperator): fetch_games>, insert_games already registered for DAG: nba_data_pipeline
[2024-10-11T21:49:50.527+0000] {logging_mixin.py:137} INFO - [2024-10-11T21:49:50.527+0000] {taskmixin.py:205} WARNING - Dependency <Task(PythonOperator): insert_games>, fetch_games already registered for DAG: nba_data_pipeline
[2024-10-11T21:49:50.527+0000] {logging_mixin.py:137} INFO - [2024-10-11T21:49:50.527+0000] {taskmixin.py:205} WARNING - Dependency <Task(PythonOperator): fetch_stats>, insert_player_stats already registered for DAG: nba_data_pipeline
[2024-10-11T21:49:50.528+0000] {logging_mixin.py:137} INFO - [2024-10-11T21:49:50.528+0000] {taskmixin.py:205} WARNING - Dependency <Task(PythonOperator): insert_player_stats>, fetch_stats already registered for DAG: nba_data_pipeline
[2024-10-11T21:49:50.534+0000] {processor.py:753} INFO - DAG(s) dict_keys(['nba_data_pipeline']) retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T21:49:50.624+0000] {logging_mixin.py:137} INFO - [2024-10-11T21:49:50.624+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2024-10-11T21:49:50.977+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 1.087 seconds
[2024-10-11T21:50:21.685+0000] {processor.py:153} INFO - Started process (PID=114) to work on /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T21:50:21.686+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/nba_data_pipeline.py for tasks to queue
[2024-10-11T21:50:21.686+0000] {logging_mixin.py:137} INFO - [2024-10-11T21:50:21.686+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T21:50:22.330+0000] {logging_mixin.py:137} INFO - [2024-10-11T21:50:22.330+0000] {taskmixin.py:205} WARNING - Dependency <Task(PythonOperator): fetch_teams>, insert_teams already registered for DAG: nba_data_pipeline
[2024-10-11T21:50:22.330+0000] {logging_mixin.py:137} INFO - [2024-10-11T21:50:22.330+0000] {taskmixin.py:205} WARNING - Dependency <Task(PythonOperator): insert_teams>, fetch_teams already registered for DAG: nba_data_pipeline
[2024-10-11T21:50:22.332+0000] {logging_mixin.py:137} INFO - [2024-10-11T21:50:22.331+0000] {taskmixin.py:205} WARNING - Dependency <Task(PythonOperator): fetch_players>, insert_players already registered for DAG: nba_data_pipeline
[2024-10-11T21:50:22.332+0000] {logging_mixin.py:137} INFO - [2024-10-11T21:50:22.332+0000] {taskmixin.py:205} WARNING - Dependency <Task(PythonOperator): insert_players>, fetch_players already registered for DAG: nba_data_pipeline
[2024-10-11T21:50:22.333+0000] {logging_mixin.py:137} INFO - [2024-10-11T21:50:22.333+0000] {taskmixin.py:205} WARNING - Dependency <Task(PythonOperator): fetch_games>, insert_games already registered for DAG: nba_data_pipeline
[2024-10-11T21:50:22.333+0000] {logging_mixin.py:137} INFO - [2024-10-11T21:50:22.333+0000] {taskmixin.py:205} WARNING - Dependency <Task(PythonOperator): insert_games>, fetch_games already registered for DAG: nba_data_pipeline
[2024-10-11T21:50:22.334+0000] {logging_mixin.py:137} INFO - [2024-10-11T21:50:22.334+0000] {taskmixin.py:205} WARNING - Dependency <Task(PythonOperator): fetch_stats>, insert_player_stats already registered for DAG: nba_data_pipeline
[2024-10-11T21:50:22.334+0000] {logging_mixin.py:137} INFO - [2024-10-11T21:50:22.334+0000] {taskmixin.py:205} WARNING - Dependency <Task(PythonOperator): insert_player_stats>, fetch_stats already registered for DAG: nba_data_pipeline
[2024-10-11T21:50:22.346+0000] {processor.py:753} INFO - DAG(s) dict_keys(['nba_data_pipeline']) retrieved from /opt/airflow/dags/nba_data_pipeline.py
[2024-10-11T21:50:22.403+0000] {logging_mixin.py:137} INFO - [2024-10-11T21:50:22.402+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2024-10-11T21:50:22.458+0000] {logging_mixin.py:137} INFO - [2024-10-11T21:50:22.458+0000] {dag.py:3441} INFO - Setting next_dagrun for nba_data_pipeline to 2023-11-01T00:00:00+00:00, run_after=2023-11-02T00:00:00+00:00
[2024-10-11T21:50:22.699+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/nba_data_pipeline.py took 1.016 seconds
